compose.yml

```diff
@@ -3,7 +3,7 @@ networks:
 
 services:
   db:
-    image: mysql:latest
+    image: mysql:9.0.1
```

Comment:
version locking is great idea.

Not necessary by any means, but most images allow you the granular control to lock on to specific major, minor, or patch versions. Instead of locking onto `mysql:9.0.1`, you could also do `mysql:9` or `myql:9.0` which would keep you up to date on the latest minor and patch versions respectively without risking jumping major versions

Reply:
https://hub.docker.com/_/mysql/tags

Reply:
Although it looks like it is not an option for `grafan/loki` or `grafana/grafana` 😢